devito/symbolics/inspection.py

```diff
@@ -189,6 +192,8 @@ def _(expr, estimate, seen):
     flops, flags = _estimate_cost.registry[object](expr, estimate, seen)
     if {S.One, S.NegativeOne}.intersection(expr.args):
         flops -= 1
+    if ImaginaryUnit in expr.args:
+        flops *= 2
```

Comment:
For all ops *=2 ?

Reply:
?

Reply:
Complex arithmetic cost 2 FMA

Reply:
So, is it guranteed that all args in an expr will be Imaginary?
Am I seeing sth wrong?